msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AuthScheme.java

```diff
@@ -0,0 +1,65 @@
+package com.microsoft.aad.msal4j;
+
+import java.net.URI;
+
+//TODO: javadocs explaining this class, link to documentation page
+public class AuthScheme {
+
+    String httpMethod;
+    URI uri;
+    String nonce;
+    SchemeType type;
+
+    public enum SchemeType {
+        POP
```

Comment:
Code smell - 1 element enum. This class isn't actually extensible to other protcols. Bearer scheme does not require HttpMethod / Uri etc.

Reply:
Removed in the latest commit